app/api/signup/views.py

```diff
@@ -0,0 +1,14 @@
+from fastapi import APIRouter
+from .schemas import Signup
+
+router = APIRouter()
+
+@router.get("/", response_model= Signup)
+async def signup_user(first_name: str,last_name: int, email: str, password: str):
+                    show = {
```

Comment:
Please use a variable descriptive of what this does. `resp` or `response` works way better here than a seemingly random `show`